### DISCUSSION TITLE
General help with pyftsubset

### DISCUSSION THREAD

**Discussion by jordan-keen** [fonttools/fonttools#2856] [Q&A] [answered]
Hi there, I'm currently trying to use pyftsubset but am not sure what steps I am doing wrong, I've installed python3, and have pip3 installed as well. Below is a screenshot of the command and error I'm getting. 

Do I need the optional dependencies? I am not able to run the command 'pip install fonttools[ufo,lxml,woff,unicode]' even when trying to use pip3.

Thank you for the help

<img width="1373" alt="Screen Shot 2022-10-19 at 11 15 10 AM" src="https://user-images.githubusercontent.com/103953368/196734321-de68b2d3-9eff-4c5b-bce3-048e9df47247.png">

**Comment by justvanrossum** [comment, marked as answer]
You need to enter the command as a single line, or escape the line breaks with backslashes.

**Reply by jordan-keen** [reply]
Thank you for the quick response! - I also was missing the extra '--' dashes
